lapce-data/src/editor/commands/test/test_state.rs

```diff
@@ -73,9 +73,12 @@ impl TestState {
             let marker_len = end - start;
 
             if whole_match.as_str().starts_with("</") {
-                if ends.insert(cursor_id, start).is_some() { panic!("Duplicate selection end marker: {whole_match:?}") }
-            } else if starts
-            .insert(cursor_id, start).is_some() { panic!("Duplicate cursor marker: {whole_match:?}") }
+                if ends.insert(cursor_id, start).is_some() {
+                    panic!("Duplicate selection end marker: {whole_match:?}")
+                }
+            } else if starts.insert(cursor_id, start).is_some() {
+                panic!("Duplicate cursor marker: {whole_match:?}")
+            }
```

Comment:
One thing I just noticed: these panics are basically assert statements. Do you think they would look better like that?

Reply:
To be honest I have no idea about it :slightly_smiling_face:

Reply:
That might have been a slight hint to try it, but all right :)